### PR TITLE
Explorer transaction mock

### PR DESCRIPTION
**Pull request by stella3d**
Here's a single mock transaction's details to unblock explorer front-end work.

The included typescript types try to account for the fact that we are in flux between using number-array encoding of bytes and using `base64` encoding.  If they differ from what's in the included json, use what's in the json.  

I'll get some more examples soon.

### CODE REVIEW

File: explorer/client/src/utils/internetapi/mock_transactions.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[
```
Comment (Andrew47) [inline explorer/client/src/utils/internetapi/mock_transactions.json:1]: Given this is static JSON data, the directory `explorer/client/src/utils/static` would be a more suitable directory. The `findDataFromID` function in `src/utils/static/searchUtil` can then be updated to search across the multiple different static JSON files.

File: explorer/client/src/utils/static/movecall-tx-response.json [hunk 2]
```diff
@@ -0,0 +1,244 @@
+{
```
Comment (Andrew47) [inline explorer/client/src/utils/static/movecall-tx-response.json:1]: This JSON file is in the correct directory; although `findDataFromID` will need to tap into this JSON file also.

**Review by Andrew47 — CHANGES_REQUESTED** [review]
This is a good start. I particularly like the idea of splitting the static JSON data across multiple files. We'll need `findDataFromID` to be updated to tap into these various JSON datasets. Perhaps if there were a script that pulled from across JSON files and exported a single data item?

File: explorer/client/src/utils/internetapi/TransactionTypes.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+type EmptySignInfo = null;
+type ClientSignedTransaction = TransactionEnvelope<EmptySignInfo>;
+
+interface CertifiedTransaction {
+    digest: SuiBytes,                   // transaction_digest
+    checked: boolean,                   // is_checked
+    tx: ClientSignedTransaction,        // transaction
+    sigs: AuthorityNameSigPair[]        // signatures
```
Comment (666lcz) [inline explorer/client/src/utils/internetapi/TransactionTypes.ts:48]: Why do we want to change the field names here? This makes deserialization harder and more error prone

File: explorer/client/src/utils/internetapi/TransactionTypes.ts [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+interface TransactionEffectsEnvelope<S> {
+    effects: TransactionEffects,
+    auth_signature?: S
+}
+
+type AuthorityName = string;
+type AuthorityNameSigPair = [AuthorityName, SuiBytes];
+
+type TransactionEnvelope<S> = {
+    is_checked: boolean,
```
Comment (666lcz) [inline explorer/client/src/utils/internetapi/TransactionTypes.ts:35]: `is_checked` is marked as `skip` [here](https://github.com/MystenLabs/sui/blob/f9ef1783083bc5011951ae3a779bb70d27cb8335/sui_types/src/messages.rs#L333), so it seems that this field won't be sent to the client side?

**Review by 666lcz — COMMENTED** [review]
Thanks for defining all these types! I can copy & paste most of them to the formal SDK

**Review by Clay-Mysten — COMMENTED** [review]
Markdown LGTM

**Comment by stella3d** [comment]
closing this because data format has changed.  easier to redo

**Pull request closed without merging**